src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball ball = new Ball();
+
+        for (int i = 0; i < 3; i++) {
+            System.out.println(ball.toString(lottery.getRandomBall()));
```

Comment:
Here, in `System.out.println(..)` you should pass created object using lottery.getRandomBall()